WordPress/Classes/Utility/Media/MediaImageExporter.swift

```diff
@@ -136,6 +129,7 @@ class MediaImageExporter: MediaExporter {
             var writer = ImageSourceWriter(url: url, sourceUTType: type as CFString)
             if let maximumImageSize = maximumImageSize {
                 writer.maximumSize = maximumImageSize as CFNumber
+                writer.lossyCompressionQuality = 0.9 as CFNumber
```

Comment:
Should we have a constant for `0.9` somewhere up top to make it slightly easier to find / tweak this in future?